recipes/circuits/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "circuits" %}
+{% set version = "3.2" %}
+{% set sha256 = "b2fad9be2235f87480c194efc91eaeed3cb030c465d4697f1ca326ce3619bf9b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - setuptools_scm
```

Comment:
This should normally be not needed, `setuptools_scm` is a build-time only dependency.
```suggestion
```

Reply:
Hi ... Before adding it, this was giving me: `ModuleNotFoundError: No module named 'pkg_resources'` on the TEST phase. Do you have any other idea on how to solve it?

Reply:
Yes, you need just `setuptools` (without `_scm`) then.